modules/ztp-precaching-ztp-config.adoc

```diff
@@ -75,6 +75,8 @@ spec:
             - name: "ens1f0"
               macAddress: "AA:BB:CC:11:22:33"
 ----
+<1> References the cluster image set that is used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.
+<2> References the cluster image set that is used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at site level.
```

Comment:
```suggestion
<2> Specifies the cluster image set used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at the site level.
```
- Would `specifies` be more accurate than `references` here?
- Edit for minimalism